mutation_query.cc

```diff
@@ -45,6 +45,15 @@ reconcilable_result::operator==(const reconcilable_result& other) const {
     return boost::equal(_partitions, other._partitions);
 }
 
+void
+reconcilable_result::merge(schema_ptr schema, const reconcilable_result& other) {
```

Comment:
If disjointness is a precondition, it should be called merge_disjoint(), and `other` should be renamed to reflect the required order.

Reply:
Or maybe the function name should reflect the order.

Reply:
Will rename.